ota-plus-web/app/reactapp/src/layouts/Main.jsx

```diff
@@ -194,11 +194,11 @@ class Main extends Component {
         }
     }
     setTermsAccepted() {
-        this.termsAndConditionsAccepted = true;
-        Cookies.set('termsAccepted', 1);
+        this.userStore.acceptContract('terms');
```

Comment:
Rather than hardcoding `terms`, this should point to the actual HTML filename that contains the accepted terms.

Reply:
@taheris ok, but I don't have any ready html or contract content to create html by myself. So I had to make a temporary solution.